src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">datatracking</string>
```

Comment:
can prefix all of this with va_ to prevent collission with customer's keys. In case they named the same.

Reply:
va_app_name, va_key_uid, va_key_sid

Reply:
relocated  va_key_uid and va_key_in session manager